include/types.h

```diff
@@ -1,5 +1,9 @@
 #pragma once
 #include <utility>
+#include <vector>
 typedef uint32_t node_id_t;
 typedef uint64_t Node;
```

Comment:
Technically not a part of this commit, but just for my own understanding: what is the difference between Node and node_id_t?

Reply:
node_id_t is the 32 bit id of a node whereas
node_t is the 64 bit id of an edge basically

The reason node_t is here at all is because the streaming repo expects it. This will be solved in a later pull request